pages/api/check-status.ts

```diff
@@ -63,24 +68,27 @@ export const searchPassportStatusApi = async (
   const response = await fetch(url)
 
   if (response.ok) {
+    logger.info('response OK')
     const searchResult: PassportStatusesSearchResult = await response.json()
     const { GetCertificateApplicationResponse } = searchResult._embedded
 
     if (GetCertificateApplicationResponse.length === 0) {
-      res.status(404).send('Passport Status Not Found')
-      return
+      logger.error('error 404: searchResult._embedded is empty')
```

Comment:
I would log this as `debug` because this isn't an application error, and we're gonna see potentially millions of these in the log file.